modules/apps/content-dashboard/content-dashboard-web/src/main/java/com/liferay/content/dashboard/web/internal/search/request/ContentDashboardSearchContextBuilder.java

```diff
@@ -193,7 +193,7 @@ public ContentDashboardSearchContextBuilder withEnd(int end) {
 		return this;
 	}
 
-	public ContentDashboardSearchContextBuilder withSort(Sort sort) {
+	public ContentDashboardSearchContextBuilder withSort(Sort... sort) {
```

Comment:
@cgoncas, changed this for multiple sorts

Reply:
Awesome!!!